lib/src/main/java/uk/gov/di/ipv/cri/drivingpermit/library/persistence/item/DocumentCheckResultItem.java

```diff
@@ -2,11 +2,13 @@
 
 import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
 import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+import uk.gov.di.ipv.cri.common.library.annotations.ExcludeFromGeneratedCoverageReport;
 
 import java.util.List;
 import java.util.Objects;
 import java.util.UUID;
 
+@ExcludeFromGeneratedCoverageReport
```

Comment:
Why are we excluding these models from the coverage report?

Reply:
The DocumentCheckResultItem bean is located in the lib as its shared with the check and issue cred lambdas. Its exclude as the coverage thresholds is per sub project with no tests or code using it in the lib, without the annotation the coverage for the lib branch coverage drops to 0.24 vs 0.64 - this is the same approach in passport-v1 https://github.com/alphagov/di-ipv-cri-uk-passport-api/blob/main/lib/src/main/java/uk/gov/di/ipv/cri/passport/library/persistence/DocumentCheckResultItem.java